### PR TITLE
Issue 146 fix

### PR DESCRIPTION
**Pull request by sythel**
complete PR #152 before addressing/reviewing

table and graph fixes for issue #146 

tab navigation prev/next buttons now follow screen (experimental)

### CODE REVIEW

**Comment by sythel** [comment]
resolving conflicts then will request review

**Comment by sythel** [comment]
NOTE: the Normalization Dam Re-order is taking place in server.R now so that changes will work for both WSM.R and WSM_graphs_test.R

File: src/dams_mcda/WSM.R [hunk 1]
```diff
@@ -268,10 +268,10 @@ WSM <- function(RawCriteriaMatrix, NormalizedMatrix, DamsData, Decisions){
 	}
 	Ind_NormalizedMatrix[is.nan.data.frame(Ind_NormalizedMatrix)] <- 0
```
Comment (elbfox) [inline src/dams_mcda/WSM.R]: This should be 
Ind_NormalizedMatrix[2:5,6,3] <- c(1,1,1,1)#This replaces properties NaN at East Millinocket
Ind_NormalizedMatrix[1,5,3] <- 1 #This replaces damage 0 value for Remove at East Millinocket
Ind_NormalizedMatrix[1,1,2] <- 1 #This  fish habitat NaN at Medway
Ind_NormalizedMatrix[5,3,1:3] <- 1#This replaces the reservoir storage NaN at West Enfield, Medway, East Millinocket
Ind_NormalizedMatrix[1,2,7] <- 1 #This replaces the river rec NaN at Millinocket Lake

Comment (sythel) [reply src/dams_mcda/WSM.R]: fixed

**Comment by sythel** [comment]
testing local before merging/pushing to live server

**Pull request merged**